pkgs/top-level/python-packages.nix

```diff
@@ -8360,9 +8360,17 @@ in {
 
   pytools = callPackage ../development/python-modules/pytools { };
 
-  pytorch = callPackage ../development/python-modules/pytorch {
-    cudaSupport = pkgs.config.cudaSupport or false;
-  };
+  pytorch =
+    let
+      cudnn = pkgs.cudnn_8_3_cudatoolkit_11;
+      cudatoolkit = cudnn.cudatoolkit;
```

Comment:
IIUC `pkgs.cudnn_8_3_cudatoolkit_11` will pull in cudatoolkit_11_5 and `cudnn.cudatoolkit` will pull in `cudatoolkit_10_1`. Just as a heads up.

This is still strictly better than broken, but worth noting that it still does pull in multiple CUDA versions AFAIU

Reply:
Wait, what o_0

Reply:
IIUC cudnn currently points to cudnn_7_6_cudatoolkit_10_1

Reply:
Check out the diff in https://github.com/NixOS/nixpkgs/pull/166784

Reply:
oh wait... maybe I'm being an idiot. I guess this is like a let-rec binding so `cudnn` refers to the definition just prior. ok in that case nvm!

Reply:
😓 I thought for a moment the same, but

```nix
nix-repl> pkgs.cudnn_8_3_cudatoolkit_11.cudatoolkit.version
"11.5.0"
```